sdcm/nemesis.py

```diff
@@ -2616,7 +2616,9 @@ def check_ssl_reload_log(node_system_log):
     def steady_state_latency(self, sleep_time=None):
         if not sleep_time:
             sleep_time = self.cluster.params.get('nemesis_interval') * 60
```

Comment:
please make it twice the nemesis_interval, so we will have longer steady_state.
Please not that it may mean to extend the duration of the test and/or the stress.

Reply:
2 options here:
1st, i can pass the `sleep_time` to the `steady_state_latency` to be 60, so no need to change anything in that logic

2nd, increase the `nemesis_interval`, as we are set now to run 1 single nemesis cycle, so even if we set in this test `nemesis_interval` to be 45 or even 60, it won't change anything else.

WDYT?

Reply:
also, this PR is to add `InfoEvent` messages to the steady state, and nothing else, to increase the `sleep_time` i will send a new PR.

Reply:
why not just:
```suggestion
            sleep_time = self.cluster.params.get('nemesis_interval') * 60 * 2
```

Reply:
why do we need 60 minutes of steady state though?

Reply:
I thought you said it's 15 mins.
60 mins could give a stabler result, but if it's 30 you can leave it as is for now.